index.html

```diff
@@ -7,6 +7,6 @@
   <title>Github - Test Commands</title>
 </head>
 <body>
-  
+  <h2>Aula 02 - Como o merge funciona.</h2>
```

Comment:
Novo título para o site.
Minha sugestão de exemplo:
```suggestion
  <h1>Aula 02 - Como o merge funciona.</h1>
```